File: Frontend/src/app/Models/ServerSendsRoomConfigurations.ts
import {BaseDto} from "./baseDto";

export class ServerSendsRoomConfigurations extends BaseDto<ServerSendsRoomConfigurations>{
  minTemparature? : number
  maxTemparature? : number
  maxHumidity? : number
  minHumidity? : number
  minAq? : number
  maxAq? : number
}

File: api/Dtos/ServerDTOs/ServerSendsRoomConfigurations.cs

using lib;

namespace socketAPIFirst.Dtos;

public class ServerSendsRoomConfigurations : BaseDto
{
    public double minTemparature { get; set; }
    public double maxTemparature { get; set; }
    public double maxHumidity { get; set; }
    public double minHumidity { get; set; }
    public double minAq { get; set; }
    public double maxAq { get; set; }

}